### PR TITLE
Sam/Fixups

### PR DESCRIPTION
**Pull request by samholmes**

### CODE REVIEW

File: src/v2/routes/putStore.ts [hunk 1]
```diff
@@ -36,10 +36,10 @@ export const putStoreRouter = (appState: AppState): Router => {
       timestamp
     })
 
-    // Send response
-    res.status(201).json(
-      makeApiResponse<PutStoreResponse>({ timestamp })
-    )
+    // Response:
+
```
Comment (swansontec) [inline src/v2/routes/putStore.ts:40]: Extra newline here?

**Review by swansontec — CHANGES_REQUESTED** [review]
Also, can you make another commit that creates `test/server.test.ts`:

```js
apiSuite('Component: Basic server errors', (appState: AppState) => {
  const app = makeServer(appState)
  const agent = supertest.agent(app)

  it('Can send 404 errors', async () => {
    await agent
      .get('/api/nowhere')
      .send()
      .expect(res => isErrorResponse(404, 'not found')(res))
  })
})
```

File: src/config.ts [hunk 2]
```diff
@@ -18,10 +20,11 @@ const isDev = NODE_ENV === 'dev'
 export type Config = ReturnType<typeof asConfig>
 
 export const asConfig = asObject({
+  couchUri: asOptional(
+    asString,
+    `http://${COUCH_USERNAME}:${COUCH_PASSWORD}@${COUCH_HOSTNAME}:${COUCH_PORT}`
```
[the remark this answers is not in the document]
Comment (samholmes) [reply src/config.ts:25]: That's the plan. Env is only used at `yarn configure` step (first install). If changes are required in the config, then this is accomplished with the existing config file.

File: src/v2/routes/putStore.ts (hunk 1 above)
Comment (samholmes) [reply src/v2/routes/putStore.ts:40]: It was to make it more consistent with the other route files. The made up convention here is that a comment on its own line (not contiguous to code) and ending with a `:` is a sort of section label. Purely my own style

**Review by swansontec — APPROVED** [review]

**Pull request merged**